frontend/leaderboard/src/Metrics.re

```diff
@@ -109,9 +113,11 @@ let calculateSnarkFeeCount = (map, block: Types.NewBlock.data) => {
              switch (feeCount) {
              | Some(feeCount) =>
                let result =
-                 Int64.add(
-                   Int64.of_string(snarkJob.fee),
-                   Int64.of_string(feeCount),
+                 convertNanoToBase(
+                   Int64.add(
+                     Int64.of_string(snarkJob.fee),
+                     Int64.of_string(feeCount),
+                   ),
```

Comment:
fwiw I think the map could be from name->int64, rather than a string, as long as the incoming snarkJob.fee is converted from a string.